jwala-services/src/main/java/com/cerner/jwala/service/binarydistribution/impl/BinaryDistributionServiceImpl.java

```diff
@@ -199,21 +201,31 @@ public boolean remoteFileCheck(final String hostname, final String remoteFilePat
         return result;
     }
 
+    /**
+     * Distrubute unzip.exe for windows and unzip.sh for other OS
+     * @param hostname host name
+     */
     @Override
     public void distributeUnzip(String hostname) {
         LOGGER.info("Start deploy unzip for {}", hostname);
         final String jwalaScriptsPath = ApplicationProperties.getRequired(PropertyKeys.REMOTE_SCRIPT_DIR);
         if (!remoteFileCheck(hostname, jwalaScriptsPath)) {
             remoteCreateDirectory(hostname, jwalaScriptsPath);
         }
-
         if (!remoteFileCheck(hostname, jwalaScriptsPath + "/" + UNZIPEXE)) {
             final String unzipFile = ApplicationProperties.get(PropertyKeys.LOCAL_JWALA_BINARY_DIR) + "/" + UNZIPEXE;
             LOGGER.info("SCP {} " + unzipFile);
             remoteSecureCopyFile(hostname, unzipFile,  jwalaScriptsPath + "/" + UNZIPEXE);

```

Comment:
I know you didn't change this code but it's still relevant: does this work in linux? In the jwala-tomcat-linux.zip there is no unzip.exe in data/binaries so I would expect this to fail.